e2e-test/src/test/resources/pluginParameters.properties

```diff
@@ -0,0 +1,15 @@
+projectId=cdf-athena
+dataset=test_automation
+
+## ERROR-COLLECTOR-PROPERTIES-START
+ecErrorMessageColumnName=messaging
+ecErrorCodeColumnName=coding
+ecErrorEmitterNodeName=staging
+defaultMsg=msg
+defaultCode=code
+defaultNode=node
+## ERROR-COLLECTOR-PROPERTIES-END
+
+## WRANGLER-PROPERTIES-START
+conditionalData=send-to-error id < 13 \nsend-to-error value < 110
```

Comment:
where are loading input data from (which table, and where is that table created)?